studio/completion_service/completion_service.py

```diff
@@ -0,0 +1,297 @@
+import os
+import subprocess
+import uuid
+import logging
+import time
+import pickle
+import tempfile
+import signal
+import re
+import six
+
+from studio import runner, model, fs_tracker
```

Comment:
I feel like we should be consistent with our relative imports (`from studio import runner` vs. `from . import runner`)

Reply:
but that's in a different folder. We could use `from .. import runner`, but we are already doing 
`from studio import something` in the tests